### PR TITLE
Narrow columns for checkboxen, statuses

### PR DESCRIPTION
**Pull request by seaninryan**
Quick PR to make checkbox and status columns have width 30

### CODE REVIEW

File: ui/src/components/volumes/VolumeListView.js [hunk 1]
```diff
@@ -80,6 +80,8 @@ class VolumeListView extends React.Component {
   render() {
     const { selected, loading, volumes } = this.state;
 
+console.log(volumes);
```
Comment (ehazlett) [inline ui/src/components/volumes/VolumeListView.js]: Probably can remove the debug.

**Review by ehazlett — COMMENTED** [review]
Minor nit other lgtm

**Comment by ehazlett** [comment]
ping @tombee

**Review by tombee — APPROVED** [review]

**Comment by tombee** [comment]
LGTM, thanks! :)

File: ui/src/components/volumes/VolumeListView.js (hunk 1 above)
Comment (seaninryan) [reply ui/src/components/volumes/VolumeListView.js]: Done

**Review by tombee — APPROVED** [review]

**Pull request merged**